content/blog/2023/08/28/2023-08-28-renovate-bote-probe-blog.adoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: Renovate Probe
+tags:
+- gsoc
+- gsoc2023
+- healthscore
+- probes
+- plugin
+authors:
+- jagruti
+opengraph:
+  image: /images/gsoc/2023/jagruti/renovate-probe.jpg
+sig:
+- gsoc
+---
+
+== Introduction
+
+Bots are used to automate tasks in a plugin.
+The Plugin Health Scoring System tracks usage of two bots - link:https://github.com/dependabot[Dependabot] and link:https://docs.renovatebot.com/[Renovate].
+
+Dependabot looks for dependencies that are outdated in a plugin. It can also raise a security alert if a security breach is found like when the code depends on an insecure package.
+
+Renovate provides automated dependency updates. It can also schedule PRs, and customize behavior using configuration files.
```

Comment:
```suggestion
Renovate provides automated dependency updates.
It can also schedule PRs and customize behavior using configuration files.
```
suggestion for formatting